scripts/jenkins/qa_openstack.sh

```diff
@@ -168,8 +168,10 @@ $zypper --gpg-auto-import-keys -n ref
 # deinstall some leftover crap from the cleanvm
 $zypper -n rm --force 'python-cheetah < 2.4'
 
-# deinstall cloud-init and dependencies
+# deinstall cloud-init and dependencies (but keep cloud-final systemd unit around)
```

Comment:
I guess, people will argue that it is not a bug, but a feature.
And I think the default is 90s.